app/uk/gov/hmrc/economiccrimelevyregistration/services/RegistrationValidationService.scala

```diff
@@ -362,6 +363,7 @@ class RegistrationValidationService @Inject() (clock: Clock, schemaValidator: Sc
         case Some(Charity)                   => validateCharity(registration)
         case Some(UnincorporatedAssociation) => validateUnincorporatedAssociation(registration)
         case Some(Trust)                     => validateTrust(registration)
+        case Some(NonUKEstablishment)        => validateNonUkEstablishment(registration)
         case _                               => ???
```

Comment:
We do not have any Other entity type left. `OtherEntityType` is sealed trait so it's safe to assume that there can't be any other type passed here for matching, that is not already been covered with previous case clauses. You can remove this line.

Reply:
Removed